pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -778,6 +772,7 @@ def test_multi_index_parse_dates(all_parsers, index_col):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.filterwarnings("ignore::FutureWarning")
```

Comment:
I wouldn't ignore warnings, you can *assert* the warnings

Reply:
Done